tests/test_slogpy.py

```diff
@@ -1,8 +1,73 @@
 """The test for the slogpy package."""
-
+import pytest
+import re
+from slogpy import slog
 from slogpy import __version__
-
-
+from slogpy.slog import Slog as slog
+from unittest.mock import patch
+import unittest
 def test_version():
     """The most basic of tests."""
     assert __version__ == "0.1.0"
+
+def test_logging_info(capsys):
+
+    slog.info("Log something at the info level")
+    captured = capsys.readouterr()
+     # Remove ANSI escape codes using regex
+    cleaned_output = re.sub(r'\x1b\[[0-9;]*m', '', captured.out)
```

Comment:
For most people regex's are not readable, even with regex101 this one is not understandable.  This one is likely to strip the content that rich adds to get colors.  It is also duplicated (cut-n-paste) many times.  This should be set as a file level constant that has a meaningful name.